config.json

```diff
@@ -34,7 +34,7 @@
       "description": "MsBuild building options.",
       "valueType": "passThrough",
       "values": [],
-      "defaultValue": "/nologo /maxcpucount /verbosity:minimal"
+      "defaultValue": "/nologo /maxcpucount /verbosity:minimal /nodeReuse:false"
```

Comment:
>/nodeReuse:false [](start = 63, length = 16)

Does this run in both Windows and non-Windows? If so then this will fail since `nodeReuse` was removed from .NetCore MSBuild